packages/permissionless/actions/bundler/getUserOperationReceipt.ts

```diff
@@ -123,10 +115,11 @@ export const getUserOperationReceipt = async <
             blockNumber: BigInt(log.blockNumber),
             blockHash: log.blockHash,
             transactionHash: log.transactionHash,
-            logIndex: BigInt(log.logIndex),
-            transactionIndex: BigInt(log.transactionIndex),
+            logIndex: Number(log.logIndex),
+            transactionIndex: Number(log.transactionIndex),
             address: log.address,
-            topics: log.topics
+            topics: log.topics,
+            removed: log.removed
```

Comment:
Why not return the log as it is?

Reply:
log might return in Hex right? As other bundlers may send in some other format